client/index.html

```diff
@@ -2,10 +2,12 @@
 <html>
   <head>
     <script src="//d2wy8f7a9ursnm.cloudfront.net/bugsnag-3.min.js" data-apikey="609d1f020472985e992a1d3d920f66eb"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/css/materialize.min.css">
     <title> One-Laptop </title>
   </head>
   <body>
     <div id="app"></div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/js/materialize.min.js"></script>
```

Comment:
I have imported it locally. It'll reflect with my PR. Is that okay with you @andela-jomadoye?